karma.conf.ts

```diff
@@ -65,7 +70,7 @@ if (
   )
 }
 
-const setting = Object.assign({}, base, override)
+const setting = objectAssign({}, base, override)
```

Comment:
`Object.assign({}, base, override)` is work, because `karma.conf.(js|ts)` works nodejs not browser .

Reply:
oh, sorry, My thinking didn't enough. thanks.